### PR TITLE
feat(format-scripts): linewrap predicates

### PR DESCRIPTION
**Pull request by lucario387**
Readability improvements no.2 :thinking:

Question: What should be the value for `textwidth`? I set it to 100 right now

Edit: `"format-ignore".kick()`

### CODE REVIEW

**Comment by clason** [comment]
100 sounds good to me. Could make it configurable, but then we'd have to add flag handling to the argument parsing logic; not sure that's worth it.

**Comment by lucario387** [comment]
Guess I'm keeping it then

**Comment by clason** [comment]
We use 100 in `stylua.toml`, so there's precedent.

File: queries/arduino/highlights.scm [hunk 1]
```diff
@@ -1,74 +1,31 @@
...
-     "highByte"
-     "lowByte"
-     ; External Interrupts
-     "attachInterrupt"
-     "detachInterrupt"
-     ; Interrupts
-     "interrupts"
-     "noInterrupts"))
+    ; Digital I/O
+    "digitalRead" "digitalWrite" "pinMode"
```
Comment (lucario387) [inline queries/arduino/highlights.scm:5]: Still readable I think @ObserverOfTime

File: queries/bash/highlights.scm [hunk 2]
```diff
@@ -106,7 +106,8 @@
 
 ; trap -l
 ((word) @constant.builtin
-  (#match? @constant.builtin "^SIG(HUP|INT|QUIT|ILL|TRAP|ABRT|BUS|FPE|KILL|USR[12]|SEGV|PIPE|ALRM|TERM|STKFLT|CHLD|CONT|STOP|TSTP|TT(IN|OU)|URG|XCPU|XFSZ|VTALRM|PROF|WINCH|IO|PWR|SYS|RTMIN([+]([1-9]|1[0-5]))?|RTMAX(-([1-9]|1[0-4]))?)$"))
+  (#match? @constant.builtin
+    "^SIG(HUP|INT|QUIT|ILL|TRAP|ABRT|BUS|FPE|KILL|USR[12]|SEGV|PIPE|ALRM|TERM|STKFLT|CHLD|CONT|STOP|TSTP|TT(IN|OU)|URG|XCPU|XFSZ|VTALRM|PROF|WINCH|IO|PWR|SYS|RTMIN([+]([1-9]|1[0-5]))?|RTMAX(-([1-9]|1[0-4]))?)$"))
```
Comment (clason) [inline queries/bash/highlights.scm:110]: Is the newline here intended?

File: queries/awk/highlights.scm [hunk 3]
```diff
@@ -9,11 +9,15 @@
 
 ; https://www.gnu.org/software/gawk/manual/html_node/Auto_002dset.html
 ((identifier) @constant.builtin
-  (#any-of? @constant.builtin "ARGC" "ARGV" "ARGIND" "ENVIRON" "ERRNO" "FILENAME" "FNR" "NF" "FUNCTAB" "NR" "PROCINFO" "RLENGTH" "RSTART" "RT" "SYMTAB"))
+  (#any-of? @constant.builtin
+    "ARGC" "ARGV" "ARGIND" "ENVIRON" "ERRNO" "FILENAME" "FNR" "NF" "FUNCTAB" "NR" "PROCINFO"
+    "RLENGTH" "RSTART" "RT" "SYMTAB"))
```
Comment (lucario387) [inline queries/awk/highlights.scm:14]: Just for clarity, I'd argue reading this is easier than
```
(#any-of? @constant.builtin "FOO" BAR"
  "BAZ" ..............................
  "ABC")
```

File: queries/bash/highlights.scm (hunk 2 above)
Comment (ObserverOfTime) [inline queries/bash/highlights.scm:110]: Might as well replace this with `any-of` while we're at it.
```suggestion
  (#any-of? @constant.builtin
        "SIGHUP" "SIGINT" "SIGQUIT" "SIGILL" "SIGTRAP" "SIGABRT" "SIGBUS" "SIGFPE" "SIGKILL" "SIGUSR1"
    "SIGSEGV" "SIGUSR2" "SIGPIPE" "SIGALRM" "SIGTERM" "SIGSTKFLT" "SIGCHLD" "SIGCONT" "SIGSTOP"
    "SIGTSTP" "SIGTTIN" "SIGTTOU" "SIGURG" "SIGXCPU" "SIGXFSZ" "SIGVTALRM" "SIGPROF" "SIGWINCH"
    "SIGIO" "SIGPWR" "SIGSYS" "SIGRTMIN" "SIGRTMIN+1" "SIGRTMIN+2" "SIGRTMIN+3" "SIGRTMIN+4"
    "SIGRTMIN+5" "SIGRTMIN+6" "SIGRTMIN+7" "SIGRTMIN+8" "SIGRTMIN+9" "SIGRTMIN+10" "SIGRTMIN+11"
    "SIGRTMIN+12" "SIGRTMIN+13" "SIGRTMIN+14" "SIGRTMIN+15" "SIGRTMAX-14" "SIGRTMAX-13" "SIGRTMAX-12"
    "SIGRTMAX-11" "SIGRTMAX-10" "SIGRTMAX-9" "SIGRTMAX-8" "SIGRTMAX-7" "SIGRTMAX-6" "SIGRTMAX-5"
    "SIGRTMAX-4" "SIGRTMAX-3" "SIGRTMAX-2" "SIGRTMAX-1" "SIGRTMAX"))
```

Comment (ObserverOfTime) [reply queries/bash/highlights.scm]: Whoops, extra space.

**Review by ObserverOfTime — APPROVED** [review]

**Review by clason — APPROVED** [review]

**Pull request merged**